src/Libraries/RevitNodes/Elements/InternalUtilities/ElementWrapper.cs

```diff
@@ -189,6 +190,17 @@ public static ScheduleView Wrap(Autodesk.Revit.DB.ViewSchedule view, bool isRevi
             return ScheduleView.FromExisting(view, isRevitOwned);
         }
 
+        public static Element Wrap(Autodesk.Revit.DB.View view, bool isRevitOwned)
+        {
+            switch (view.ViewType)
+            {
+                case ViewType.Legend:
+                    return Legend.FromExisting(view, isRevitOwned);
+                default:
+                    return UnknownElement.FromExisting(view, true);
```

Comment:
why not pass isRevitOwned here as well?

Reply:
Good catch.

Reply:
Although that's what you all did for all other `UnknownElement` switch cases. Look few lines up at `ViewPlan` wrapper. The default is using `true` instead of the `isRevitOwned`